batch7crud-roll4/src/main/webapp/resources/js/PostForm.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by pratishshr on 2/1/16.
+ */
+
+function PostForm() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this 'PostForm' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AS100)